src/helper/config.cpp

```diff
@@ -60,6 +60,7 @@ const std::unordered_map<std::string_view, std::string_view> config::defaults =
         {"lock_path", "/tmp/ddappsec.lock"},
         {"socket_path", "/tmp/ddappsec.sock"},
         {"log_level", "info"},
+        {"runner_idle_timeout", "2880" } // minutes
```

Comment:
2 days seems excessive. You'd need a server that gets no requests for 2 days. I think this could be safely changed to something much lower, like half an hour. The only penalty is like a couple hundred milliseconds of delay once a new request comes in.

Reply:
As discussed, lets make this 1 day.